frontend/src/components/ShootStatus.vue

```diff
@@ -41,14 +41,14 @@ limitations under the License.
       </v-tooltip>
     </div>
     <ansi-text v-if="!!popperMessage" :text="popperMessage"></ansi-text>
-    <template v-if="isError">
-      <v-divider class="my-2"></v-divider>
-      <h4 class="error--text text-xs-left">Last Error</h4>
-      <template v-for="errorCodeDescription in errorCodeDescriptions">
+    <v-divider class="my-2"></v-divider>
+    <h4 class="error--text text-xs-left">Last Errors</h4>
+    <div v-for="lastErrorDescription in lastErrorDescriptions" v-bind:key="lastErrorDescription.description">
```

Comment:
as key: use hash? or irrelevant?

Reply:
We could generate a random value to identify the messages and add it to the errorDescription Object to overcome this. I assumed this to be a hypothetical problem but you are right, it _could_ happen and lead to a warning / out-of-sync dom. I will take care of it.